OpenRA.Game/Sound/Sound.cs

```diff
@@ -69,7 +69,8 @@ ISoundSource LoadSound(string filename)
 						return soundEngine.AddSoundSourceFromMemory(rawData, channels, sampleBits, sampleRate);
 				}
 
-				throw new InvalidDataException(filename + " is not a valid sound file!");
```

Comment:
Apparently not in https://github.com/OpenRA/OpenRA/pull/10482/files#diff-798c8d09024b0e9c0ff1bdb93edd2abcR56 above. It is stupid to constantly crash the RA2 mod for weeks now, just because button click sounds are not parsed correctly.